CONTRIBUTING.md

```diff
@@ -36,6 +36,25 @@ This subfolder contains some example `.rst` files that show how to implement spe
 3. Run `pip install -e .`
 4. Run `sphinx-build -b html example_docs/docs/ example_docs/docs/_build/html` to build the documentation. This will create the `example_docs/docs/_build` subfolder where you can see the html files generated based on the `.rst` files in the `docs` folder and the styles pulled from the `qiskit_sphinx_theme`.
 
+------
+## Updating bundled web components
+
+We use web components from https://github.com/Qiskit/web-components to include common design elements from qiskit.org in our documentation.
+
+To update the top nav bar web component:
+
+1. In https://github.com/Qiskit/web-components, run `npm install` then `npm run build`.
+2. There should be a file created at the root of the repository called `experimental-bundled-ui-shell.js`. Copy its contents into the file `js/web-components/top-nav-bar.js`. 
+3. Build the example docs to ensure everything works.
+
+If you want to add a new web component:
+
+1. Work with the web components repository to set up `npm run build` to generate a single JavaScript file.
+2. Add the file contents to the `js/web-components/` folder.
```

Comment:
also unclear here which commands need to be run in the web components or sphinx theme repo